natgeo/mortar
# [DO-NOT-MERGE][MORTAR-187] Validate compatibility for Pestle 1.0 to 2.0

*Please don't merge this code*

To test:

- ```gulp serve```
- When ```gulp serve``` open the page, click on "Components" on the top menu, you should see the slider section on it.
- You can verify both Pestles are running and loading components writing on the console:
** ```NGS.platform.getInitializedComponents()``` for pestle v1
** ```Pestle2.ModuleManager.getAllInstances()``` for pestle v2
